troposphere/emr.py

```diff
@@ -156,7 +156,7 @@ class SimpleScalingPolicyConfiguration(AWSProperty):
     props = {
         'AdjustmentType': (basestring, False),
         'CoolDown': (positive_integer, False),
-        'ScalingAdjustment': (positive_integer, True),
+        'ScalingAdjustment': (integer, True),
```

Comment:
Given this can be specified as a percentage (docs indicate -0.20 for a 20% decrease), perhaps this should be a float instead. Also, extra credit to add a validator that looks at the AdjustmentType (if specified). For example, EXACT_CAPACITY should be a positive integer.